### ISSUE TITLE
Ligatures don't render in MacOS terminal

### ISSUE THREAD

**Issue by bebraw** [MonoLisaFont/feedback#41]
I tested MonoLisa against MacOS default terminal. Renders below:

**Fira Code**

<img width="88" alt="Screenshot 2020-10-17 at 12 20 11" src="https://user-images.githubusercontent.com/166921/96334753-b80fd800-107b-11eb-9256-d3fb3aada797.png">

**MonoLisa**

<img width="99" alt="Screenshot 2020-10-17 at 12 20 51" src="https://user-images.githubusercontent.com/166921/96334763-c100a980-107b-11eb-8ae4-d95e98c9e690.png">

MacOS terminal doesn't have any specific setting to enable/disable ligatures. The question is, what's causing the difference and should this be fixed?

**Comment by okonet** [comment]
Confirming this. Not sure how I didn't notice this before...

**Comment by bebraw** [comment]
Given both Fira and MonoLisa have been authored using Glyphs, I imagine it's some difference in output settings or perhaps ligature groups. Reference for comparison: https://github.com/tonsky/FiraCode .

**Comment by MarcusSterz** [comment]
@mekkablue Can you have a look?

**Comment by bebraw** [comment]
I think the difference might be due to https://github.com/tonsky/FiraCode/issues/854 . It seems FiraCode is using `calt` (contextual alternates) for these so we would have to move the ligatures to that group to have them enabled everywhere by default.

**Comment by okonet** [comment]
I'm not sure how `call` and ligatures are related (afaik they are not) but since https://github.com/MonoLisaFont/website/issues/55 is still open I guess we also use `calt`. @MarcusSterz any comments?

**Comment by okonet** [comment]
Apparently Terminal only support `calt` so starting from v1.600 all ligatures from `calt` (i.e. non-symbols) will be visible in Terminal. Unfortunately we can't do much about symbols from `liga` and because the separation of those is a "feature" we won't act on this one anymore.

**Comment by alessandrodn** [comment]
What's does it mean in practice? I installed version v1.600 and I don't see any difference...

**Comment by okonet** [comment]
It means that support for ligatures is limited it Terminal and you will only see a subset called contextual alternatives defined in `calt` as per https://www.monolisa.dev/specimen

**Comment by repoles** [comment]
So, there is still no way to enable MonoLisa ligatures in macOS default terminal, like it does when using Fire Code or JetBrains Mono?

**Comment by bebraw** [comment]
@repoles Did you try freezing the ligatures with the customize tool? If this does not work, please open a new issue.

**Comment by repoles** [comment]
@bebraw how can I freeze the ligatures? Are you mean customize the font before download? If so I didn't see any option to freeze ligatures. The only option I changed was to use script variant.
<img width="508" alt="Screenshot 2023-01-19 at 12 01 57" src="https://user-images.githubusercontent.com/1004592/213476735-98a37666-e580-4d6b-8e02-37347bd08af5.png">

**Comment by bebraw** [comment]
@repoles It's the top-right. I see it's not possible to freeze them based on that. Please open a separate issue so we don't lose the feature request.

**Comment by okonet** [comment]
No, it’s not possible and can’t be possible actually. I’d recommend iTerm2 which has support for ligatures.

**Comment by okonet** [comment]
BTW it’s only `Liga` that’s not working. `calt`works in Terminal out of the box but there is no way to change these settings AFAIK.

**Comment by repoles** [comment]
I don't know nothing about how font ligature works. And I know macOS Terminal support for this feature is weak. I tried iTerm2 and other terminals, but keep preferring the basic, simple, efficient, out-of-the-box Terminal ;) But since ligatures work properly on this app when I'm using `Fira Code` or `JetBrains Mono`, I thought maybe there was a way to use the same approach to make `MonoLisa` work the same. But this is not a big deal, really. The font is simply beautiful and I'm happy to be able to use it with ligatures in my code editors.

**Comment by okonet** [comment]
Hmm, I need to check Fira Code with my Terminal.

**Comment by davedice** [comment]
I have the same problem on MacOS.  Ligatures work as expected in iTerm2 (which I temporarily installed and removed just to test the feature), in vscode, in mvim, etc., but not in terminal.app, which is where I really wanted the feature.   Other ligaturized fonts, such as Jetbrains, BerkeleyMono, Julia, Cascadia, Fira, etc., all work as expected with visible ligatures in terminal.app.  

There's a comment in https://github.com/MonoLisaFont/feedback/issues/191 about augmenting customizer to allow "liga" to be merged into "calt", which would be interesting try.     I took a quick look at fontforge and fontfreeze, but didn't seen any obvious way to try that myself.
